src/server.ts

```diff
@@ -27,12 +27,12 @@ export default function runServer({ port = 8080 } = { port: 8080 }) {
 	console.log('server starting');
 
 	// TODO: do this in a watch
```

Comment:
Can we also create an issue in this repository for this? I think it will remain better visible as an issue instead of just a comment in the code

Reply:
https://fontoxml.atlassian.net/browse/DEV-6727